cmd/server/command/root.go

```diff
@@ -34,6 +49,7 @@ func NewCommand() (*cobra.Command, error) {
 	command.PersistentFlags().StringVar(&options.GrafanaDevUrl, "grafana-dev-url", os.Getenv("GRAFANA_DEV_URL"), "grafana dev url")
 	command.PersistentFlags().StringVar(&options.GrafanaStagingUrl, "grafana-staging-url", os.Getenv("GRAFANA_STAGING_URL"), "grafana staging url")
 	command.PersistentFlags().StringVar(&options.GrafanaProdUrl, "grafana-prod-url", os.Getenv("GRAFANA_PROD_URL"), "grafana prod url")
+	command.PersistentFlags().StringSliceVar(&users, "user-mappings", []string{}, "user mappings between github and slack, key-value pair: <github>=<slack>")
```

Comment:
Maybe mention that the mapping is of emails. Further more, it can be used for what ever committer info, not just GitHub, right?

Reply:
That's true. Good point!